docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-114.adoc

```diff
@@ -0,0 +1,47 @@
+== Clarifai API Key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|TBD
+
+|Checkov Check ID
+|CKV_SECRET_114
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+A Clarifai API Key is vital for accessing Clarifai's powerful AI and machine learning services, which provide advanced image and video recognition capabilities. This key is crucial for authenticating requests to the API and ensuring that only authorized clients can utilize the services. Exposure of this key can lead to unauthorized access, misuse of the API, and potential data leaks. Securely managing this key is essential to maintain the security and integrity of your applications. Regularly reviewing your security settings and practices is important to prevent unauthorized usage. For detailed security guidelines and best practices, refer to the latest https://docs.clarifai.com/clarifai-basics/authentication/personal-access-tokens/[Clarifai documentation and support] resources.
+
+=== Fix - Buildtime
+
+*Clarifai*
+
+If your Clarifai API Key is exposed, immediate action is required to revoke the compromised key and generate a new one to secure access to the API. Follow these steps to replace the exposed key and update all dependent applications:
```

Comment:
If your Clarifai API key is exposed, it’s crucial to take immediate action to prevent any unauthorized use. Regenerate a new key through your Clarifai account and replace the exposed key with the new key in all relevant applications. By regenerating a compromised Clarifai API Key and updating your application integrations, you not only secure your access to Clarifai AI services but also maintain the confidentiality and integrity of your data flows.